src/eko/evolution_operator/grid.py

```diff
@@ -34,8 +34,8 @@ class OperatorGrid:
             configuration dictionary
```

Comment:
Even in this file, above, there is an absolute `import` from `eko`, please make it relative (as all the others)